### PR TITLE
NC001 Add Register Smoke Test

### PR DESCRIPTION
**Pull request by ghost**
Creating smoke test for Register page

### CODE REVIEW

File: src/main/java/com/nopcommerce/demo/pages/RegisterPage.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+
+
+    @CacheLookup
+    @FindBy(xpath = "//a[normalize-space()='Register']")
+    WebElement verifyRegisterLink ;
+
+    public void verifyRegisterLink(){
+
```
Comment (bhavika48) [inline src/main/java/com/nopcommerce/demo/pages/RegisterPage.java:24]: if possible dont leave this much gap

Comment (ghost) [reply src/main/java/com/nopcommerce/demo/pages/RegisterPage.java:24]: resolve

**Review by HiteshVagQa — APPROVED** [review]

**Review by ghost — COMMENTED** [review]
fix

**Review by ghost — COMMENTED** [review]
fix

**Pull request merged**